src/sections/fragenAntworten.html

```diff
@@ -0,0 +1,101 @@
+                </div>
+            </div>
+            
+            <div class=" grid gap-16 sm:mt-10 sm:flex sm:gap-10 md:gap-24">
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+            </div>
+            
+            <div class=" grid gap-16 sm:mt-10 sm:flex sm:gap-10 md:gap-24">
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+            </div> 
+        </div>
+    
+        <div class="mt-20 lg:mt-36 xl::mt-44 2xl:mt-64 pb-36">
+            <h1 class="ml-5 md:ml-16 lg:ml-28 xl:ml-36 2xl:ml-0 font-railwayBold text-base sm:text-xl md:text-2xl lg:text-3xl">Nehmen Sie Kontakt zu uns auf!</h1>
+            <a href="#">
+                <div class="flex items-center gap-2 sm:gap-5 bg-yellow-500 ml-4 md:ml-12 lg:ml-24 xl:ml-32 2xl:ml-0 pl-3 sm:px-7 py-2 lg:py-3 w-56 sm:w-72 md:w-80 rounded-full mt-2 sm:mt-4 md:mt-8">
+                    <svg width="27" height="20" viewBox="0 0 27 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M23.8506 0.5C25.1709 0.5 26.2881 1.61719 26.2881 2.9375C26.2881 3.75 25.8818 4.46094 25.2725 4.91797L14.2529 13.1953C13.6436 13.6523 12.8818 13.6523 12.2725 13.1953L1.25293 4.91797C0.643555 4.46094 0.288086 3.75 0.288086 2.9375C0.288086 1.61719 1.35449 0.5 2.72559 0.5H23.8506ZM11.3076 14.5156C12.4756 15.3789 14.0498 15.3789 15.2178 14.5156L26.2881 6.1875V16.75C26.2881 18.5781 24.8154 20 23.0381 20H3.53809C1.70996 20 0.288086 18.5781 0.288086 16.75V6.1875L11.3076 14.5156Z" fill="#001342"/>
+                    </svg>
```

Comment:
It's an ideal idea to not include the SVG path in your code in this manner. The reason for that is it adds unnecessary complexity. Instead, it would be a good idea to create an external file, perhaps in the assets folder, and place the SVG path there. You can then include it as an image using the <img> tag. This approach will help reduce the complexity of the codebase.